python/lsst/meas/deblender/baseline.py

```diff
@@ -204,7 +203,7 @@ def __init__(self, filterName, footprint, maskedImage, psf, psffwhm, avgNoise,
         # avgNoise is an estiamte of the average noise level for the image in this filter
         if avgNoise is None:
             stats = afwMath.makeStatistics(self.varimg, self.mask, afwMath.MEDIAN)
-            avgNoise = math.sqrt(stats.getValue(afwMath.MEDIAN))
+            avgNoise = np.sqrt(stats.getValue(afwMath.MEDIAN))
```

Comment:
Agreed, but at this point it seems like a lot of trouble for such a trivial change. Would you be ok with not worrying about splitting the commits and leaving it as is? If so, this branch is ready to merge.